.travis.yml

```diff
@@ -7,17 +7,18 @@ addons:
   postgresql: "9.3"
 env:
   matrix:
-    - MODULE=!querydsl-scala
-    - MODULE=querydsl-scala
+    - MODULE=!querydsl-scala PROFILES=all,travis,examples
+    - MODULE=!querydsl-jpa-codegen PROFILES=jpa,hibernate5,travis,examples
```

Comment:
I thought it would run more than necessary, but it doesn't have `all` so it's fine.
Carry on :wink: